docs/api/js/accounts-l1-l2.md

```diff
@@ -225,19 +225,19 @@ const abi = [
 ];
 const contractInterface = new ethers.utils.Interface(abi);
 const calldata = contractInterface.encodeFunctionData("increment", []);
-const ergsLimit = BigNumber.from(1000);
+const l2gasLimit = BigNumber.from(1000);
 
 const txCostPrice = await wallet.getBaseCost({
   gasPrice,
   calldataLength: ethers.utils.arrayify(calldata).length,
-  ergsLimit,
+  l2gasLimit,
 });
 
 console.log(`Executing the transaction will cost ${ethers.utils.formatEther(txCostPrice)} ETH`);
 
 const executeTx = await wallet.requestExecute({
```

Comment:
please, update it so it reflects https://github.com/matter-labs/zksync-web-v2-docs/pull/274/files#r1098706461 these changes

Reply:
you did not